templates/dice/tooltip.hbs

```diff
@@ -29,4 +29,9 @@
     </div>
   </section>
   {{/each}}
+  <ul>
+  {{#each breakdown}}
+    <li>{{{this}}}</li>
+  {{/each}}
+  </ul>
```

Comment:
It is worth to add a condition to avoid an empty `<ul></ul>` if the breakdown array is unused.
```suggestion
  {{if breakdown.length}}
    <ul>
    {{#each breakdown}}
      <li>{{{this}}}</li>
    {{/each}}
    </ul>
  {{/if}}
```
(did not test this change)